app/controllers/callbacks_controller.rb

```diff
@@ -8,6 +8,19 @@ def google_oauth2
       return
     end
     @user = User.from_omniauth(request.env["omniauth.auth"])
+    make_admin @user
     sign_in_and_redirect @user
   end
+
+  # Check admins file from project root and make user admin if listed there
+  def make_admin(user)
```

Comment:
can we name the method something like `check_if_admin(user)` ? `make_admin` kinda implies that it would always make a person admin, imo